proxy/src/coco/peer.rs

```diff
@@ -10,6 +10,7 @@ pub use librad::net::peer::{PeerApi, PeerConfig};
 use librad::uri::RadUrn;
 use radicle_surf::vcs::git::{self, git2};
 
+use crate::discovery::{get_feed, Feed};
```

Comment:
This goes against our dependency direction, where inner modules (i.e. protocol wrappers like `coco` and `registry`) are consumed by outer ones (e.g. `project`). There should be a way to wrap whatever coco offfers into our domain understanding. An example for this kind of exercise is `coco::source`, it's easy to imagine a `coco::feed` module to handle this concern.

Reply:
So does the import on `project::Project` below.

Reply:
Cool cool makes sense